code/controllers/subsystem/mapping.dm

```diff
@@ -71,6 +72,9 @@ SUBSYSTEM_DEF(mapping)
 		if(!config || config.defaulted)
 			to_chat(world, "<span class='boldannounce'>Unable to load next or default map config, defaulting to Box Station</span>")
 			config = old_config
+	if(map_adjustment)
+		map_adjustment.job_change()
+		log_world("Applied '[map_adjustment.map_file_name]' map adjustment: on_mapping_init()")
```

Comment:
you should run the job change at the end of SSJob init.

Reply:
I am not sure SSJob is initialized before mapping subsystem.
map adjustment is loaded first then it runs a proc for each needed point for every subsystem is initialized. I believe this throws runtime.

Reply:
`log_world("Applied '[map_adjustment.map_file_name]' map adjustment: on_mapping_init()")`
well, you took a wrong proc.

Reply:
ah, then it must be because of PreInit().
map adjustment should be loaded quickly before the subsystem runs Init()